python/mdvtools/socketio_upload.py

```
@@ -0,0 +1,714 @@
+                            last_activity_dt = state.get('last_activity_time') 
+                            if status in ['uploading', 'resuming', 'error', 'queued', 'processing'] and last_activity_dt:
+                                if isinstance(last_activity_dt, datetime): 
+                                    if now - last_activity_dt > self.upload_ttl:
+                                        upload_log(f"Upload {file_id} expired, cleaning up")
+                                        self._delete_upload_dir(file_id)
+                        else:
+                            # Orphaned directory
+                            try:
+                                dir_mod_time = datetime.fromtimestamp(os.path.getmtime(item_path), timezone.utc)
+                                if now - dir_mod_time > self.upload_ttl:
+                                     upload_log(f"Cleaning up orphaned directory: {item_path}")
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)